payjoin/src/receive/v2/mod.rs

```diff
@@ -60,6 +61,36 @@ pub struct Receiver {
     context: SessionContext,
 }
 
+/// A wrapper around the receiver session. The receiver session is accessible only after it has been persisted via `persist`.
```

Comment:
you can use ``[`persist`]`` to link to the function in rustdoc

Reply:
Done